src/lp/process.c

```diff
@@ -159,51 +183,51 @@ static inline void silent_execution(const struct lp_ctx *lp, array_count_t last_
 
 /**
  * @brief Send anti-messages
- * @param proc_p the message processing data for the LP that has to send anti-messages
+ * @param lp the message processing data for the LP that has to send anti-messages
  * @param past_i the index in @a proc_p of the last validly processed message
  */
-static inline void send_anti_messages(struct process_ctx *proc_p, array_count_t past_i)
+static inline void send_anti_messages(struct process_ctx *lp, const array_count_t past_i)
```

Comment:
Are you sure you want to name the first argument `lp`? We use `lp` throughout the codebase for variables of type `struct lp_ctx *`, while this is an inner component of type `struct process_ctx`. I agree that `proc_p` is ugly, maybe use `pp`?

Reply:
Silly me, I confused the argument by mistake.